tests/groupTest.js

```diff
@@ -32,19 +32,22 @@ describe('Main: Currently testing group management,', function () {
         headers: xmlResponseHeaders,
         body: new XmlBuilder('1.0', '', 'ocs').build(ocs => {
           ocs.appendElement('meta', '', (meta) => {
-            return ocsMeta(meta, 'ok', '100')
+            return ocsMeta(meta, 'ok', '100', MatchersV3.regex('(OK)?', ''))
```

Comment:
does the client need this response, if not we should ignore it altogether and never send it back

Reply:
It seems that client needs `meta.message` too. If we are to append `message` element only if needed like this
```js
if (Message !== null) {
    meta.appendElement('message', '', Message)
}
```
then the consumer tests will fail with following error
```js
TypeError: Cannot convert undefined or null to object
        at Function.keys (<anonymous>)

      392 |       ret = meta.message
      393 | 
    > 394 |       if (Object.keys(meta.message).length === 0) {
          |                  ^
      395 |         // no error message returned, return the whole message
      396 |         ret = json
      397 |       }
```
At least empty `message` element needs to be appended